### DISCUSSION TITLE
Stream To Telegram Channel

### DISCUSSION THREAD

**Discussion by Tabarane** [bluenviron/mediamtx#3934] [General]
Hi , i have been long time user for MediaMTX , i want to try to stream to Telegram Channel

Telegram provide us with 2 things to receive a stream which is
ServerIP / Server Token in below format
rtmps://dc4-1.rtmp.t.me/s/     -> Server IP
121345678910:qwertyuioplkjhGFDSA   -> StreamKey

so considering i have 2 USB Devices and i can fetch them from MediaMTX with different format , how i can send the streams to Telegram Channel?

**Comment by aler9** [comment]
Hello, what you want is forwarding streams from MediaMTX to an external server. This is explained in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#forward-streams-to-other-servers

just replace `-f rtsp rtsp://other-server:8554/another-path` with `-f flv rtmp://telegram-url/s/stream-key`.
